.travis.yml

```diff
@@ -11,11 +11,26 @@ compiler:
     - clang
 
 env:
-    - {}
+# Seems like Python2 doesn't work well on Focal.
+# Should be uncommented if someone fixes that
+#    - {}
```

Comment:
Focal I think is Py3 by default; so not likely going to get fixed very well. Probably have to add the extra Py2 specific packages.

We may want to keep the primary/base-OS at bionic until we move to Py3 support being required. $0.02

Reply:
There are other distros that do not have python2 support. Fedora 32, for example. It doesn't seem to support python2 **at all**.

Reply:
In other words, it looks like 0.6.x will need to be limited to those distros that do support py2, and 0.7.x will need to be truly a mixed py2-py3 release. At least the way I see it.

Reply:
Agreed on that, and that is exactly what I have done here. Focal compiles Py3 only, and Bionic compiles both